internal/util/k8sutil/k8sutil.go

```diff
@@ -120,11 +120,11 @@ func GetTypeMetaFromBytes(b []byte) (t metav1.TypeMeta, err error) {
 // dns1123LabelRegexp defines the character set allowed in a DNS 1123 label.
 var dns1123LabelRegexp = regexp.MustCompile("[^a-zA-Z0-9]+")
 
-// FormatOperatorNameDNS1123 ensures name is DNS1123 label-compliant by
+// FormatStringDNS1123 ensures a string is DNS1123 label-compliant by
 // replacing all non-compliant UTF-8 characters with "-".
-func FormatOperatorNameDNS1123(name string) string {
-	if len(validation.IsDNS1123Label(name)) != 0 {
-		return dns1123LabelRegexp.ReplaceAllString(name, "-")
+func FormatStringDNS1123(s string) string {
+	if len(validation.IsDNS1123Label(s)) != 0 {
+		return dns1123LabelRegexp.ReplaceAllString(s, "-")
 	}
-	return name
+	return s
 }
```

Comment:
I agree 100% with the change. But, should we not mark it as breaking change?

Reply:
This is an internal utility.

Reply:
It is true. It is in the internal. 👍

Reply:
Reverted, will do in a followup.